lua/entities/turret_40mm_frag/init.lua

```diff
@@ -74,17 +73,23 @@ function ENT:Think()
 
             return true
         end
-
+        
+        -- damage equals 400 multiplied by two thirds of this turret's firing speed
+        local baseDamage = 185

```

Comment:
Maybe you could calculate that here so you don't need the comment?

Reply:
I tried hardcoding the damage calculations but it felt like too much, if we wanted to change it later it would've been really weird.